man/systemd.network.xml

```diff
@@ -594,6 +594,17 @@
         </para></listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>ProxyNDP=</varname></term>
```

Comment:
hmm, so far we prefixed IPv6-specific functionality with "IPv6", i.e. maybe this should better be called "IPv6ProxyNDP="?